hasql.cabal

```diff
@@ -89,6 +89,7 @@ library
     text >= 1 && < 2,
     bytestring >= 0.10 && < 0.11,
     hashable >= 1.2 && < 1.3,
+    network-ip >= 0.2 && < 1,
```

Comment:
I used 0.3 version locally with GHC 8.0.2. Version 0.2 is necessary for older GHC versions. So l will change upper bound to 0.4